doctorapp/src/components/home/Sidebar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="personal-information">
+    <img :src="img" alt="profile picture" />
+    <h1>{{ name }}</h1>
+    <p><strong>Description:</strong> {{ description.slice(0, 150) }} ...</p>
+    <p><strong>Email:</strong> <a :href="'mailto:' + mbox">{{ mbox.replace('mailto:', '') }}</a></p>
+    <p><strong>Phone Number:</strong> <a :href="'tel:' + phone">{{ phone.replace('tel:', '') }}</a></p>
+  </div>
```

Comment:
Just for sementic HTML sake's, this `div` can be replaced with an `aside` tag, as it contain the aside content of the page. You can find more info [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)
```suggestion
  <aside class="personal-information">
    <img :src="img" alt="profile picture" />
    <h1>{{ name }}</h1>
    <p><strong>Description:</strong> {{ description.slice(0, 150) }} ...</p>
    <p><strong>Email:</strong> <a :href="'mailto:' + mbox">{{ mbox.replace('mailto:', '') }}</a></p>
    <p><strong>Phone Number:</strong> <a :href="'tel:' + phone">{{ phone.replace('tel:', '') }}</a></p>
  </aside>
```